tests/Unit/Countries/US/Oregon/V20190101/EugeneTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\Oregon\V20190101;
+
+use Appleton\Taxes\Countries\US\Oregon\Eugene\Eugene;
+use Appleton\Taxes\Tests\Unit\Countries\TaxTestCase;
+use Appleton\Taxes\Tests\Unit\Countries\TestParameters;
+use Appleton\Taxes\Tests\Unit\Countries\TestParametersBuilder;
+
+class EugeneTest extends TaxTestCase
+{
+    private const DATE = '2019-07-09';
+    private const EUGENE_LOCATION = 'us.oregon.eugene';
+    private const OREGON_LOCATION = 'us.oregon';
+    private const ALABAMA_LOCATION = 'us.alabama';
+    private const TAX_CLASS = Eugene::class;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->query_runner->addTax(self::TAX_CLASS);
+    }
+
+    /**
+     * @dataProvider provideTestData
+     */
+    public function testEugeneTax(TestParameters $parameters): void
+    {
+        $this->validate($parameters);
+    }
+
+    /**
+     * @dataProvider provideTestDataOutOfArea
+     */
+
+    public function testEugeneTaxOutOfArea(TestParameters $parameters): void
+    {
+        $this->disableTestQueryRunner();
+        $this->validate($parameters);
+    }
+
+    public function provideTestData(): array
+    {
+        $builder = new TestParametersBuilder();
+        $builder
+            ->setDate(self::DATE)
+            ->setTaxClass(self::TAX_CLASS)
+            ->setPayPeriods(52);
+
+        return [
+            '00' => [
+                $builder
+                    ->setHomeLocation(self::OREGON_LOCATION)
+                    ->setWorkLocation(self::EUGENE_LOCATION)
+                    ->setWagesInCents(1125)
+                    ->setMinutesWorked(60)
+                    ->setExpectedAmountInCents(null)
```

Comment:
this will probably break when you merge master in, it will expect `0` now instead of `null`